cli/tests/test_git_fetch.rs

```diff
@@ -1206,6 +1212,125 @@ fn test_git_fetch_some_of_many_bookmarks(subprocess: bool) {
     }
 }
 
+fn create_change_id_repo(work_dir: &TestWorkDir) -> String {
```

Comment:
nit: just inline this function. It doesn't seem generally useful, and there's only one caller.